### PR TITLE
Doc: Image.memory only accepts compressed format

### PR DESCRIPTION
**Pull request by liyuqian**
See https://github.com/flutter/flutter/issues/37423

### CODE REVIEW

**Review by Piinks — APPROVED** [review]
LGTM!

File: packages/flutter/lib/src/widgets/image.dart [hunk 1]
```diff
@@ -533,6 +533,10 @@ class Image extends StatefulWidget {
   ///
   /// The [bytes], [scale], and [repeat] arguments must not be null.
   ///
+  /// This only accepts compressed image formats (e.g., PNG). Uncompressed
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/image.dart]: remove the comma after e.g.?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/image.dart (hunk 1 above)
Comment (liyuqian) [reply packages/flutter/lib/src/widgets/image.dart]: Done. Thanks!

**Pull request merged**